src/parsing.jl

```diff
@@ -305,4 +305,31 @@ function replace_dotted!(e, membernames)
     x_new = replace_syms!(e.args[1], membernames)
     y_new = protect_replace_syms!(e.args[2], membernames)
     Expr(:., x_new, y_new)
+end
+
+"""
+    fix_args(args...; )
+
+Given multiple arguments which can be any type
+of expression-like object (`Expr`, `QuoteNode`, etc.),
+appends them
```

Comment:
"appends them"? AFAICT this collects them in a vector after removing line numbers?

Also, couldn't the function use a more specific name than "fix"?

Reply:
fixed.